theforeman.org/pipelines/release/pipelines/katello.groovy

```diff
@@ -43,7 +43,6 @@ pipeline {
             agent { label 'admin && sshkey' }
 
             steps {
-                push_rpms_katello(katello_version)
```

Comment:
can we make this `if katello_version  in ['3.17', '3.18', '4.0'] { push_rpms_katello(katello_version) }` (or whatever the right syntax is)

as 3.17/3.18 def need to remain updateable on f-people and 4.0 I'd prefer to keep in sync for the release cycle

Reply:
Done.